src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/OrganizationState.scala

```diff
@@ -0,0 +1,113 @@
+package ch.epfl.bluebrain.nexus.admin.organizations
+import java.time.Instant
+import java.util.UUID
+
+import cats.effect.Async
+import ch.epfl.bluebrain.nexus.admin.config.AppConfig.HttpConfig
+import ch.epfl.bluebrain.nexus.admin.config.Vocabulary.nxv
+import ch.epfl.bluebrain.nexus.admin.organizations.OrganizationCommand._
+import ch.epfl.bluebrain.nexus.admin.organizations.OrganizationEvent._
+import ch.epfl.bluebrain.nexus.admin.types.{ResourceF, ResourceMetadata}
+import ch.epfl.bluebrain.nexus.commons.types.identity.Identity
+
+/**
+  * Enumeration of organization states.
+  */
+sealed trait OrganizationState extends Product with Serializable
+
+object OrganizationState {
+
+  /**
+    * The initial (undefined) state.
+    */
+  final case object Initial extends OrganizationState
```

Comment:
if you define it like:
```
sealed trait Initial extends OrganizationState
final case object OrganizationState extends OrganizationState
```
it allows pattern matching on the type without having to suffix the object with `.type`

Reply:
that's wrong, I meant:
```
sealed trait Initial extends OrganizationState
final case object Initial extends Initial
```